File: bootapp/src/main/java/sk/slsp/workshop/mazesolver/Application.java
package sk.slsp.workshop.mazesolver;

import org.camunda.bpm.spring.boot.starter.annotation.EnableProcessApplication;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@SpringBootApplication
@EnableProcessApplication("mySimpleApplication")
@EnableJpaRepositories(basePackages = {"com.example.test01"})
public class Application {
	public static void main(String[] args) {
		SpringApplication.run(Application.class, args);
	}
}

File: bootapp/src/main/kotlin/sk/slsp/workshop/mazesolver/Application.kt
package sk.slsp.workshop.mazesolver

import org.camunda.bpm.spring.boot.starter.annotation.EnableProcessApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.jpa.repository.config.EnableJpaRepositories

@SpringBootApplication
@EnableProcessApplication("mySimpleApplication")
@EnableJpaRepositories(basePackages = ["sk.slsp.workshop.mazesolver"])
class Application

fun main(args: Array<String>) {
	runApplication<Application>(*args)
}
